AppKit/CPTableHeaderView.j

```diff
@@ -456,34 +459,26 @@ var CPTableHeaderViewResizeZone = 3.0,
     _activeColumn = -1;
 }
 
-- (void)mouseEntered:(CPEvent)theEvent
-{
-    var location = [theEvent globalLocation];
-
-    if (CGPointEqualToPoint(location, _mouseEnterExitLocation))
-        return;
-
-    _mouseEnterExitLocation = location;
+@end
 
-    [self _updateResizeCursor:theEvent];
-}
+@implementation CPTableHeaderView (CPTrackingArea)
 
-- (void)mouseMoved:(CPEvent)theEvent
+- (void)updateTrackingAreas
 {
-    [self _updateResizeCursor:theEvent];
+    [self removeAllTrackingAreas];
+    
+    var options = CPTrackingCursorUpdate | CPTrackingActiveInKeyWindow;
+    
+    for (var i = 0; i < _tableView._tableColumns.length; i++)
+        [self addTrackingArea:[[CPTrackingArea alloc] initWithRect:[self _cursorRectForColumn:i]
+                                                           options:options
+                                                             owner:self
+                                                          userInfo:nil]];
 }
 
-- (void)mouseExited:(CPEvent)theEvent
+- (void)cursorUpdate:(CPEvent)anEvent
 {
-    var location = [theEvent globalLocation];
-
-    if (CGPointEqualToPoint(location, _mouseEnterExitLocation))
-        return;
-
-    _mouseEnterExitLocation = location;
-
-    // FIXME: we should use CPCursor push/pop (if previous currentCursor != arrow).
-    [[CPCursor arrowCursor] set];
+    [self _updateResizeCursor:anEvent];
```

Comment:
The first line of _updateResizeCursor says the code at the beginning can be removed when we have tracking rects, and now we do, so you can remove it.